scripts/gen_vimdoc.py

```diff
@@ -1096,15 +1096,13 @@ def main(config, args):
 
         docs = ''
 
-        i = 0
-        for filename in CONFIG[target]['section_order']:
+        for i, filename in enumerate(CONFIG[target]['section_order']):
             try:
                 title, helptag, section_doc = sections.pop(filename)
             except KeyError:
                 msg(f'warning: empty docs, skipping (target={target}): {filename}')
                 msg(f'    existing docs: {sections.keys()}')
                 continue
-            i += 1
```

Comment:
This `i` seems unused since https://github.com/neovim/neovim/commit/985bc6c6e055785e9cee22c1fd70127f82cdbffb